cob_hardware_config/package.xml

```diff
@@ -33,7 +33,7 @@
   <exec_depend>raw_description</exec_depend>
   <exec_depend>robot_state_publisher</exec_depend>
   <exec_depend>rviz</exec_depend>
-  <exec_depend>ur_description</exec_depend>
+  <!--exec_depend>ur_description</exec_depend-->
```

Comment:
`ur_description` is not released into `melodic`...all robots using it, could stay on the `indigo_dev` branch and still use `kinetic`